css/css-values/getComputedStyle-calc-mixed-units-003.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+
+  <meta charset="UTF-8">
+
+  <title>CSS Values Test: computed value of calc() values using multiplication/division with mixed units</title>
+
+  <link rel="help" href="https://www.w3.org/TR/css-values-4/#calc-computed-value">
+
+  <meta name="flags" content="">
+  <meta content="This meta-test checks for the resolution and absolutization of computed values with mixed units to 'px' when multiplication/division is used." name="assert">
+
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+
+  <style>
+  html {
+      font-size: 30px;
+  }
+
+  body {
+      font-size: 16px;
+      line-height: 1.25; /* computed value: 20px */
+      height: 500px;
+      margin: 20px;
+      width: 520px;
+  }
+
+  div#target {
+      height: 100px;
+      width: 100px;
+  }
+  </style>
+
+  <div id="target"></div>
+
+  <script>
+  function startTesting() {
+    var targetElement = document.getElementById("target");
+
+    function verifyComputedStyle(property_name, specified_value, expected_value, description) {
+      test(function() {
+        targetElement.style.setProperty(property_name, "initial");
+        targetElement.style.setProperty(property_name, specified_value);
+        assert_equals(getComputedStyle(targetElement)[property_name], expected_value);
+      }, description);
+    }
+
+    verifyComputedStyle("width", "calc(10% * 8.4 + 2em)", "84px", "testing width: calc(10% * 8.4 + 2em)");
```

Comment:
As the comment below says, the expected value should be `468px`? I don't get where the `84px` comes from. And ditto for other cases below.

Reply:
Whoops I think I left then `84px` from an old test I copied. Thanks

Resolved in [2a682d8](https://github.com/web-platform-tests/wpt/pull/42517/commits/2a682d8b77a35036c0a71b538fda4730a963fc6d)